understanding/on-input.html

```diff
@@ -58,85 +56,73 @@
-               <header class="box-h ">Page Contents</header>
+               <header class="box-h ">このページの内容</header>
                <div class="box-i">
                   <ul>
-                     <li><a href="#brief">In Brief</a></li>
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 3.2.2:</span>On Input (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 3.2.2:</span>入力時 (レベル A)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>Changing the setting of any <a href="#dfn-user-interface-component">user interface component</a> does not automatically cause a <a href="#dfn-changes-of-context">change of context</a> unless the user has been advised of the behavior before using the component.
+                  <p><a href="#dfn-user-interface-component">ユーザインタフェース コンポーネント</a>の設定を変更することが、<a href="#dfn-changes-of-context">コンテキストの変化</a>を自動的に引き起こさない。ただし、利用者が使用する前にその挙動を知らせてある場合を除く。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Content can be operated more predictably.</dd>
+                  <dt>目標</dt>
+                  <dd>コンテンツをより予測可能な方法で操作できる。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Forewarn users if their context will change based on their input.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>入力内容に応じてコンテキストが変化する場合は、利用者に事前に警告する。</dd>
```

Comment:
この達成基準の趣旨からして、この input は入力された「内容」ではなく、ユーザーによる入力操作を指しているのではないでしょうか。「内容」トリでこんな感じではどうでしょうか?

```suggestion
                  <dd>利用者の入力に応じてコンテキストが変化する場合は、利用者に事前に警告する。</dd>
```